### ISSUE TITLE
Social Media Video Snippets

### ISSUE THREAD

**Issue by ahmadabugosh** [Giveth/giveth-planning#1258]
Hey team, 

For these talks, let's start the process of creating social media video snippets for them. How it works is that someone tells me what good quotes would be (timestamps of around 30 seconds that would make a good video) and I'll make them into videos we can share on social media. 

These are the talks we I'm aware of: 

1) Combating the Metacrisis
by Griff Green
(need talk link)

2) Why DAOs will change the world
by Lauren Luz
https://www.youtube.com/watch?v=CoboTd6OI-E

3) DAO it like you mean it
by Lauren Luz
(need talk link)

4) Dissecting Sustainable DAO Governance and Effective Operations
by Griff Green
(need talk link)

**Comment by ahmadabugosh** [comment]
Hey @danelsuga @clara-zi , Do either of you have some bandwidth can you take this on? 

Mainly just suggest quotes or snippets we should use from each of the talks (with timestamps)

**Comment by danelsuga** [comment]
Yes!!

**Comment by danelsuga** [comment]
Why DAOs will change the world by Lauren Luz
https://www.youtube.com/watch?v=CoboTd6OI-E
14:25 "Regenerative economics brings market feedback into the public goods sector so that we can actually have sustainable funding and innovation."

**Comment by danelsuga** [comment]
I've already done the one of Lauren's for which there is a link. I was going to paste it here but figured I'd wait for the others. Been searching for links. In touch with @laurenluz.

**Comment by danelsuga** [comment]
Still waiting for talks to be published. Here are Lauren's meanwhile if you want them now @ahmadabugosh (I've got everything in a doc but still awaiting the others):

Why DAOs will change the world
by Lauren Luz
https://www.youtube.com/watch?v=CoboTd6OI-E

3:40 “The opportunity we have here in web3 is to make a new model around public goods that replaces these inefficient systems around nonprofits and around governments providing public goods so that we actually have a chance to change the world for the better.”

14:25 “Regenerative economics brings market feedback into the public goods sector so that we can actually have sustainable funding and innovation.”

18:20 “By empowering nonprofits to become impact DAOs, we can foster community-led, economically sustainable, innovative public goods creation and can build a world of abundance that everyone can enjoy.”

**Comment by danelsuga** [comment]
@clara-zi 
https://docs.google.com/document/d/1GsPejo0Qw66UAJY23XIxFgA5Qf7vCJpgEB53ly5Qh9k/edit?usp=sharing

**Comment by ahmadabugosh** [comment]
hey @laurenluz, 

I'm adding the social media snippets to this sheet (you can download them directly from the links). So far your talk  "Why DAOs will change the world" is done, so you can post that one: https://docs.google.com/document/d/1GsPejo0Qw66UAJY23XIxFgA5Qf7vCJpgEB53ly5Qh9k/edit

**Comment by danelsuga** [comment]
@ahmadabugosh Griff's are in there now too. That was a hard one to do, as he kept cutting himself off. 😂 (And also b/c the topic was morbid much of the time!)

**Comment by laurenluz** [comment]
I highlighted and comments on the ones I think would be good for twitter @ahmadabugosh
Out of the quotes suga picked from griff's talk

**Comment by laurenluz** [comment]
I made tweets and scheduled all 3 clips from my talk FYI

**Comment by clara-zi** [comment]
Lauren's panel "DAO it like you mean it" - done. I have searched but cannot find the vid link for other panel. Anyone got it?
@laurenluz @danelsuga

**Comment by laurenluz** [comment]
@clara-zi - I don't think it is posted yet - can you keep an eye out (maybe run a search every day) and then share here if/when it's live?

**Comment by danelsuga** [comment]
@ahmadabugosh @laurenluz Griff's panel is ready in there now too.

**Comment by ahmadabugosh** [comment]
@laurenluz @clara-zi Added 3 videos for the Metacrisis talk

**Comment by danelsuga** [comment]
Do you have a link @ahmadabugosh ?

**Comment by ahmadabugosh** [comment]
They're on this sheet : https://docs.google.com/document/d/1GsPejo0Qw66UAJY23XIxFgA5Qf7vCJpgEB53ly5Qh9k/edit

I shared them under each quote I took

**Comment by danelsuga** [comment]
Oh right. 🤣 Thanks.

**Comment by clara-zi** [comment]
> can you keep an eye out (maybe run a search every day) and then share here if/when it's live?

quick update: still waiting on this one

**Comment by clara-zi** [comment]
@laurenluz @danelsuga who's the best contact for ETH Denver YouTube - I'd like to follow-up and get the link to get Lauren's outstanding panel?

**Comment by danelsuga** [comment]
I think we've exhausted this for the time being.

**Comment by c0ric0ri** [comment]
Ok @danelsuga do you want to move this to closed then? Or why is it in Backlog?

**Comment by danelsuga** [comment]
Wasn't sure if we would revisit it, if ever we could find the last video. I guess not.

**Comment by laurenluz** [comment]
no need to close it until they are all scheduled to go out
